frontend/src/reducers/story.js

```diff
@@ -7,11 +7,14 @@ import {
   FETCH_STORY_START,
   FETCH_STORY,
   INVALID_STORY,
+  SUBMIT_TRAVELS_START,
+  SUBMIT_TRAVELS,
 } from "../actions/types";
 
 const initialState = {
   status: {},
   story: null,
+  travels: [],
```

Comment:
The story now has a `travel`  field, why not using that?

Reply:
I don't want to throw too much into the `story` slice of the store. I'll do it if you feel it's worth it, but in my experience, having a single element with so much weight inside can get tricky to handle.